extensions/amp-carousel/0.1/slidescroll.js

```diff
@@ -479,4 +476,19 @@ export class AmpSlideScroll extends BaseSlides {
       this.slidesContainer_./*OK*/scrollLeft = interpolate(pos);
     }, 80, 'ease-out').thenAlways();
   }
+
+  /**
+   * Cancels the touchmove events for the element so that viewer does not
+   * consider the swipes in the carousel as swipes for changing AMP documents.
+   * @private
+   */
+  cancelTouchEvents_() {
+    // TODO(aghassemi): https://github.com/ampproject/amphtml/issues/4742
+    // prevents us from using SwipeXRecognizer with an empty handler to
+    // cancel the events. This work around is not great and temporary until
+    // #4742 is fixed.
+    this.element.addEventListener('touchmove', evt => {
+      evt.stopPropagation();
+    });
```

Comment:
Done with lint.

We need to do this regardless of `hasNativeSnapPoints_`, although the bug is very hard to reproduce in Chrome (which is `hasNativeSnapPoints_ == false`), is it still there.